content/api/suppression-list.apib

```diff
@@ -165,85 +176,58 @@ Perform a filtered search for entries in your suppression list.
-                        "type": "transactional",
-                        "source": "Bounce Rule",
-                        "description": "550: this email address does not exist #55",
-                        "created": "2015-01-01T01:01:01+00:00",
-                        "updated": "2015-01-01T01:01:01+00:00"
-                    }
-                ]
-            }
-
-## Retrieve, Delete, Insert or Update [/suppression-list/{recipient_email}]
-
-### Retrieve a Recipient Suppression Status [GET]
 
-Retrieve the suppression status for a specific recipient by specifying the recipient’s email address in the URI path.
+### Retrieve a Suppression [GET /suppression-list/{recipient_email}]
 
-This is a search endpoint so delays can occur when searching across multiple suppression lists.  See "Searching with Subaccounts" below.
+Return all suppression entries for a recipient. If the recipient is not in the suppression list, an HTTP status of 404 is returned.
 
-If the recipient is not in the suppression list, an HTTP status of 404 is returned. If the recipient is in the list, an HTTP status of 200 is returned with the suppression records in the response body. Specifying the `type` query parameter allows for retrieving only the `transactional` or `non_transactional` record. If type is specified and the recipient isn't suppressed for that type, an HTTP status of 404 is returned.
-
-In addition to the list entry attributes, the response body also includes `created` and `updated` timestamps.
-
-##### Searching with Subaccounts:
-Please provide the X-MSYS-SUBACCOUNT header when performing a lookup on a specific suppression list.
+##### Searching with Subaccounts
+If your account has subaccounts, please provide the `X-MSYS-SUBACCOUNT` header when performing a lookup on a specific suppression list.
 - Use a value of 0 to only search against the master account suppression list.
 - Use the subaccount's ID to perform a lookup on a specific subaccount suppression list.
 
-If the X-MSYS-SUBACCOUNT header is not provided, a search will be performed across the master account and all subaccount suppression lists. Searches across multiple lists can return out of date results, with a delay of up to 20 minutes. Searches against a specific list are not affected by this delay and return up-to-date information. If you do not take advantage of SparkPost's subaccounts feature, you do not need to provide the X-MSYS-SUBACCOUNT header in order to receive up-to-date results.
+If the `X-MSYS-SUBACCOUNT` header is not provided, a search will be performed across the master account and all subaccount suppression lists.
+Searches across multiple lists can return out of date results, with a delay of up to 20 minutes.
+Searches against a specific list are not affected by this delay and return up-to-date information.
+If you don't have subaccounts, you do not need to provide the X-MSYS-SUBACCOUNT header in order to receive up-to-date results.
+
++ Data Structure: Headers
```

Comment:
I think we should leave this off for the moment and then add it in across the board another time. Right now, it feels like a snowflake.

Reply:
We'll also eventually be able to put the Parameters anywhere in the content which will help with this. That way we'll be able to order it as URI, Headers, Parameters, Request Body.